test/conformance/kuttl/validate/clusterpolicy/standard/apicalls/subjectaccessreview/cm-test-ns.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cm
+  namespace: kyverno
```

Comment:
I think the reason this is passing in the test is not for the reason you may think. You're creating this ConfigMap in the `kyverno` Namespace which should be excluded by default, therefore the AdmissionReview is not seen by the admission controller. You should create this in some other Namespace (the file indicates a Namespace called `test`).